flathub.json

```diff
@@ -0,0 +1,4 @@
+{
+  "skip-arches": ["aarch64"],
```

Comment:
BTW any reason for skipping aarch64 (and not skipping 32-bit arm)?

Godot have some issues with built-it theora and vpx cross-build configuration, but with `builtin_libtheora=no` and `builtin_libvpx=no` it should be OK.

Reply:
I just copied it over from the Godot Flatpak; I don't have an ARM setup to test it, unfortunately. Is there an easy way to build a Flatpak for 32-bit x86 or ARM from a 64-bit x86 machine?

Reply:
As long as you have qemu-static-user installed, flatpak-builder should let you build other arches specified with `--arch`.

Reply:
Going to merge it as is for now, please take a look at enabling aarch64 when you have some time.